cmd/environment_variables.go

```diff
@@ -18,6 +19,23 @@ func SetFlagsFromEnvVariables(fs *pflag.FlagSet) {
 		envVar := flagToEnvVarName(f)
 		if val, present := os.LookupEnv(envVar); present {
 			fs.Set(f.Name, val)
+			return
+		}
+
+		// Do not look for _FILE if the application is already expecting a file.
+		if strings.HasSuffix(envVar, "_FILE") {
+			return
+		}
+
+		val, present := os.LookupEnv(envVar + "_FILE")
+		if present {
+			value, err := os.ReadFile(val)
+			if err != nil {
+				PrintError(errors.Wrapf(err, "failed to read file %s", envVar+"_FILE"))
```

Comment:
I think we should consider this a fatal error. Perhaps modify the function to return an error and have the callers return the error back up to `main()`?

Reply:
Made the change. It's not the most elegantly written, as `fs.VisitAll()` works by a callback rather than a loop. Let me know if you think of a better way to implement it.